src/supla/network/html/button_update.cpp

```diff
@@ -1,26 +1,47 @@
 #include "button_update.h"
-
 #include <supla/network/web_sender.h>
 
-#ifdef ARDUINO_ARCH_ESP32
-#include <HTTPUpdateServer.h>
-HTTPUpdateServer httpUpdater;
-#else
-#include <ESP8266HTTPUpdateServer.h>
-ESP8266HTTPUpdateServer httpUpdater;
-#endif
+/*
```

Comment:
Copyright and licence notice should be on first lines in file. Move it to top and put includes below.